packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1656,6 +1660,29 @@ export const components = ({
             }
           ),
 
+          ["&[data-ods-type='search']"]: {
+            borderColor: odysseyTokens.HueNeutral400,
+
+            [`& .${inputBaseClasses.input}::placeholder`]: {
+              color: odysseyTokens.TypographyColorSupport,
+              opacity: 1,
+            },
+          },
+
+          ["&[data-ods-variant='filled']"]: {
+            backgroundColor: odysseyTokens.HueNeutral50,
+            borderColor: odysseyTokens.HueNeutral50,
+
+            [`&:hover`]: {
+              borderColor: odysseyTokens.HueNeutral400,
+            },
+
+            [`&.${inputBaseClasses.focused}`]: {
+              borderColor: odysseyTokens.FocusOutlineColorPrimary,
+              boxShadow: `0 0 0 1px ${odysseyTokens.FocusOutlineColorPrimary}`,
```

Comment:
Don't we have box shadows in our design tokens?

Reply:
This isn't a true box shadow, we're using `box-shadow` to achieve the focus ring on inputs. We don't need to tokenize this whole value because this is being applied to `inputBaseClasses` and will propagate to every other place that needs it.